macos/FluentUITestViewControllers/TestColorViewController.swift

```diff
@@ -52,8 +52,9 @@ class TestColorViewController: NSViewController {
 
 		let switchButton = NSSwitch(frame: CGRect(x: 1, y: 1, width: 100, height: 50))
 		switchButton.target = self
+		switchButton.state = useColorProvider ? .on : .off
 		switchButton.action = #selector(toggleClicked)
-		toggleTextView.string = "Default"
+		toggleTextView.string = "Use ColorProvider"
```

Comment:
The problem is we now are not testing the case where the colors are overridden without a provider object, but maybe that's ok if the intent is soon to just remove the ability to do so?

To that point, should we mark usage of those setters as deprecated?

Reply:
Yeah, that was my intention since we will be deprecating this. In terms of deprecating, was going to bring that up separately after we fix up callers.

Reply:
Added a follow-up task to track this.